src/js/main.test.js

```diff
@@ -1,8 +1,8 @@
-describe('Test Suite', function(){
-    it('should pass', function(){
-        expect(true).to.equal(true);
-    });
-    it('should pass', function(){
-        assert.ok(true, "it is true");
-    })
-});
+//describe('Test Suite', function(){
+//    it('should pass', function(){
+//        expect(true).to.equal(true);
+//    });
+//    it('should pass', function(){
+//        assert.ok(true, "it is true");
+//    })
+//});
```

Comment:
This file can probably be removed at this point. It was really only necessary to get the test framework working properly. Now that we've completed that step and have some "real" tests, we can remove this.